_overviews/core/implicit-classes.md

```diff
@@ -7,6 +7,12 @@ partof: implicit-classes
 languages: [zh-cn]
 
 permalink: /overviews/core/:title.html
+versionSpecific: true
+scala2: true
```

Comment:
![Screenshot 2023-05-04 at 09 53 23](https://user-images.githubusercontent.com/13436592/236143320-3cc30d0c-62a0-4d4e-a28c-6a0b061b250f.png)

perhaps this message is too strongly worded, given that implicit classes do still exist. We could enable to customise the message that appears in this box?

Reply:
I think this is fine. I also refined the sentence below that explains how Scala 3 is different.